pootle/static/js/admin/duedates/components/DueDatePicker.css

```diff
@@ -165,12 +265,6 @@
 
 /* Modifiers */
```

Comment:
This section contained our customizations (yes, really), and everything under `/* DayPicker styles */` were stock styles. It would be great to keep that separation as it makes upgrading or switching to CSS modules a easier.